integration/tests/epochs/recover_epoch/recover_epoch_efm_test.go

```diff
@@ -19,16 +21,75 @@ type RecoverEpochSuite struct {
 	Suite
 }
 
-// TestRecoverEpoch ensures that the recover_epoch transaction flow works as expected. This test will simulate the network going
-// into EFM by taking a consensus node offline before completing the DKG. While in EFM mode the test will execute the efm-recover-tx-args
-// CLI command to generate transaction arguments to submit a recover_epoch transaction, after submitting the transaction the test will
-// ensure the network is healthy.
+// TestRecoverEpoch ensures that the recover epoch governance transaction flow works as expected and a network that
+// enters Epoch Fallback Mode can successfully recover. This test will do the following:
+// 1. Manually triggers EFM by turning off a collection node before the end of the DKG forcing the DKG to fail.
+// 2. Generates epoch recover transaction args using the epoch efm-recover-tx-args.
+// 3. Submit recover epoch transaction.
+// 4. Ensure expected EpochRecover event is emitted.
+// TODO(EFM, #6164): Currently, this test does not test the processing of the EpochRecover event
 func (s *RecoverEpochSuite) TestRecoverEpoch() {
-	s.AwaitEpochPhase(context.Background(), 0, flow.EpochPhaseSetup, 20*time.Second, time.Second)
-	fmt.Println("in epoch phase setup")
+	// 1. Manually trigger EFM
+	// wait until the epoch setup phase to force network into EFM
+	s.AwaitEpochPhase(s.Ctx, 0, flow.EpochPhaseSetup, 10*time.Second, 500*time.Millisecond)
+	// pausing collection node will force the network into EFM
+	ln := s.GetContainersByRole(flow.RoleCollection)[0]
+	_ = ln.Pause()
```

Comment:
Is it possible to write (?)
```suggestion
	require.NoError(s.T(), ln.Pause())
```